RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2798,34 +2817,24 @@ boolean providePlusNonCombat(int amt, boolean doEquips)
 		}
 		return false;
 	}
+	
+	if(forbid_extra && are_we_done()) return true;	//did equipment reach goal.
 
+	//you can toggle this effect between -combat and +combat for free without limit
 	if(have_effect($effect[Become Intensely Interested]) > 0)
 	{
 		visit_url("charsheet.php?pwd=&action=newyouinterest");
-		if(are_we_done()) return true;
+		if(forbid_extra && are_we_done()) return true;
 	}
 
+	//free to uneffect
 	foreach eff in $effects[Carlweather\'s Cantata Of Confrontation, Driving Obnoxiously]
 	{
 		uneffect(eff);
-		if(are_we_done()) return true;
-	}
-	
-	if(doEquips)
-	{
-		addToMaximize("-200combat " + to_string(-1 * amt) + "max");
-		//update our equipDiff value since we changed maximizer string
-		simMaximize();
-		equipDiff = to_int(simValue("Combat Rate") - numeric_modifier("Combat Rate"));
-		if(are_we_done()) return true;
-	}
-
-	foreach eff in $effects[Patent Invisibility]
-	{
-		buffMaintain(eff, 0, 1, 1);
-		if(are_we_done()) return true;
```

Comment:
removed as duplicate. look at line 2830 where it also does buffmaintain for patent invisibility